dockerfiles/init/modules/che/templates/che.env.erb

```diff
@@ -23,7 +22,8 @@ JPDA_ADDRESS=<%= scope.lookupvar('che::che_debug_port') %>
 
 CHE_DATABASE=/data
 CHE_WORKSPACE_STORAGE=/data/workspaces
-CHE_WORKSPACE_LOGS=/data/logs/machine
+CHE_LOGS_DIR=/logs
```

Comment:
If it is hardcoded here then it is not possible to reassign it in user's che.env. Am I correct?